pydis_site/apps/main/views/home.py

```diff
@@ -0,0 +1,106 @@
+import requests
+from django.conf import settings
+from django.shortcuts import render
+from django.utils import timezone
+from django.views import View
+
+from pydis_site.apps.main.models import RepoData
+
+GITHUB_API = "https://api.github.com/users/python-discord/repos"
+
+
+class Home(View):
```

Comment:
The other views and viewsets of the API app follow the convention `xView`, I feel `Home` is a bit undescriptive. How about renaming this to `HomeView`?

Reply:
done.